api-errors.json

```diff
@@ -12,51 +12,51 @@
         "long_message": "The request was invalid. The client should not repeat this request without modification"
     },
     "40002": {
-        "short_message": "Redirect URI mismatch",
+        "short_message": "Redirect URI Mismatch",
         "long_message": "The redirect_uri does not match the registered redirect_uri"
     },
     "40003": {
         "short_message": "Invalid Scope",
         "long_message": "An invalid or insufficient scope was used"
     },
     "40004": {
-        "short_message": "Already Registered",
+        "short_message": "User Already Registered",
         "long_message": "This user is already registered"
     },
     "40005": {
-        "short_message": "Error in phonenumber",
+        "short_message": "Invalid Phone Number",
         "long_message": "The request can't be processed due to phonenumber not meeting the required format"
     },
     "40006": {
-        "short_message": "Error in password",
+        "short_message": "Invalid Password",
         "long_message": "The request can't be processed due to password not meeting the required format"
     },
     "40007": {
-        "short_message": "Error in email",
+        "short_message": "Invalid Email",
         "long_message": "The request can't be processed due to email not meeting the required format"
     },
     "40008": {
-        "short_message": "Unprocessable Entity",
+        "short_message": "Malformed JSON Payload",
```

Comment:
This short message conflicts with `40026` as they would both correspond to the same code

Reply:
yes, you're right, I actually looked into getting rid of `40026` since it doesn't seem needed (it's fairly easy to do), but then I didn't want to make that part of this PR (and I also didn't want to artificially come up with a different short message just so that it's different). but I can make it part of this PR to remove `40026` if you think that's better.